src/components/custom-antd/Tooltip/Tooltip.tsx

```diff
@@ -1,3 +1,5 @@
+import React from 'react';
```

Comment:
Is react autoimported for you over here?

Reply:
My IDE (Webstorms) complains about relying on globally accessible imports with the UMD. I've seen these imports of React in many places in the codebase. So I guess it is safer to include it. Either way we should be concise to either include it or not in all the files in the codebase.